app/models/link.rb

```diff
@@ -58,25 +58,33 @@ def position_type
     return 'other'
   end
 
-  # The text for the short relationship link that appears on entity profile pages.
-  def link_content
-    display_date_range = relationship_label.display_date_range
-    maybe_asterisk = relationship.notes.present? ? '*' : ''
-    [
-      description,
-      display_date_range.present? ? ' ' : '',
-      display_date_range,
-      maybe_asterisk
-    ].join('')
-  end
+  concerning :Description do
+    # The text for the short relationship link that appears on entity profile pages.
+    def link_content
+      "#{description}#{link_date_range}#{relationship_notes_mark}#{ownership_stake}"
```

Comment:
Arguably this stuff should be in a presenter, but I thought best to keep it here for now. Could maybe be refactored later.

Reply:
Agreed! Much of the core profile pages were written before I learned about the presenter pattern. Refactoring these pages eventually is a good idea.